### DISCUSSION TITLE
Recently added question

### DISCUSSION THREAD

**Discussion by mgb744** [LeoKlaus/plappa#225] [Q&A]
Hi all is there any information on the recently added cab as to how long ago is considered recent? It appears that all of my audiobooks are in the recently added tab. Is the recently added tab more of a sort by latest added?

If possible, could an option be created to change What is considered recent ie last week, last month, last year?

**Comment by LeoKlaus** [comment]
Recently added shows all items in the order they have been added to your server.
It's not configurable right now, but it would be possible to add a setting for that.
